src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        randomBall.setColor(new ColorSupplier().getRandomColor().name());
+        randomBall.setNumber(new Random().nextInt(100));

```

Comment:
1. Make Random and ColorSupplier a class-level variable.
2. All magic numbers in your code should be constants.